src/devices/src/legacy/x86_64/cmos.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright 2025 Red Hat, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::cmp::min;
+
+use crate::bus::BusDevice;
+
+const INDEX_MASK: u8 = 0x7f;
+const INDEX_OFFSET: u64 = 0x0;
+const DATA_OFFSET: u64 = 0x1;
+const DATA_LEN: usize = 128;
+
+pub struct Cmos {
+    index: u8,
+    data: [u8; DATA_LEN],
+}
+
+impl Cmos {
+    pub fn new(mem_below_4g: u64, mem_above_4g: u64) -> Cmos {
+        debug!("cmos: mem_below_4g={mem_below_4g} mem_above_4g={mem_above_4g}");
+
+        let mut data = [0u8; DATA_LEN];
+
+        // Extended memory from 16 MB to 4 GB in units of 64 KB
+        let ext_mem = min(
+            0xFFFF,
+            mem_below_4g.saturating_sub(16 * 1024 * 1024) / (64 * 1024),
```

Comment:
nit: can't we use a constant for the values? The same in other places.

Reply:
Normally I would agree, but in this particular case, where we're doing math on directly on the final values, I think it's better to see the values instead of seeing constants (and then having to look for the value of those constants).